src/Impostor.Api/Net/Messages/C2S/HandshakeC2S.cs

```diff
@@ -4,6 +4,8 @@ namespace Impostor.Api.Net.Messages.C2S
 {
     public static class HandshakeC2S
     {
+        private static readonly int ExtendedHandshakeMinVersion = GameVersion.GetVersion(2021, 6, 30);
```

Comment:
Its probably going to be even more extended in future, so maybe name it `Version1` or smth